### PR TITLE
[docs] Add Input buttons and axes missing descriptions

### PR DESCRIPTION
**Pull request by abarichello**

### CODE REVIEW

File: doc/classes/Input.xml [hunk 1]
```diff
@@ -103,6 +104,7 @@
 			<argument index="0" name="axis_index" type="int">
 			</argument>
 			<description>
+				Receives a JOY_AXIS_* Enum and returns it's equivalent name as a String
```
Comment (akien-mga) [inline doc/classes/Input.xml]: Couple style changes:
"Receives a **[code]**JOY_AXIS_\*[/code] constant** and returns **its** equivalent name as a **s**tring**.**"

File: doc/classes/Input.xml [hunk 2]
```diff
@@ -111,6 +113,7 @@
 			<argument index="0" name="button" type="String">
 			</argument>
 			<description>
+				Returns the index of the provided button name
```
Comment (akien-mga) [inline doc/classes/Input.xml]: Missing dot at the end.

**Comment by aBARICHELLO** [comment]
Updated requested changes, should be ready to merge.

**Comment by akien-mga** [comment]
Thanks, looks good. Would you be able to squash the commits together? See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

**Comment by aBARICHELLO** [comment]
I think it's done now. Sorry for the trouble!

**Comment by akien-mga** [comment]
Perfect, thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**